### PR TITLE
fix: remove exclusion for Node API `azure-functions-linux`

### PR DESCRIPTION
**Pull request by janpio**
part of https://github.com/prisma/e2e-tests/issues/1748

### CODE REVIEW

File: platforms-serverless/azure-functions-linux/func-placeholder/index.ts [hunk 1]
```diff
@@ -1,7 +1,13 @@
...
+import { Prisma, PrismaClient } from '@prisma/client'
 
-const client = new PrismaClient()
+const client = new PrismaClient({
+  datasources: {
+    db: {
+      url: process.env.AZURE_FUNCTIONS_LINUX_PG_URL,
+    },
+  },
+})
```
Comment (janpio) [inline platforms-serverless/azure-functions-linux/func-placeholder/index.ts]: We should technically only do this for Node-API.

File: platforms-serverless/azure-functions-linux/finally.sh [hunk 2]
```diff
@@ -1,12 +1,30 @@
...
+
 set -eux
 
 app="$(cat func-tmp.txt)"
 
 group="prisma-e2e-linux"
+subscriptionId="63cbd994-4c44-4b0d-ba9f-463c6ec6ee31"
+
+echo "Too View Logs Visit"
+echo "https://portal.azure.com/#blade/WebsitesExtension/FunctionMenuBlade/monitor/resourceId/%2Fsubscriptions%2F${subscriptionId}%2FresourceGroups%2F${group}%2Fproviders%2FMicrosoft.Web%2Fsites%2F${app}%2Ffunctions%2F${app}"
```
Comment (janpio) [inline platforms-serverless/azure-functions-linux/finally.sh]: Will only be available if we do not delete the function and insights, so should be moved down into the  `else`.

Comment (williamluke4) [reply platforms-serverless/azure-functions-linux/finally.sh]: Moved to else

File: platforms-serverless/azure-functions-linux/func-placeholder/index.ts (hunk 1 above)
Comment (williamluke4) [reply platforms-serverless/azure-functions-linux/func-placeholder/index.ts]: Removed

**Review by williamluke4 — APPROVED** [review]

**Pull request merged**